### PR TITLE
Explain how to join vertical nav items with a bar

### PR DESCRIPTION
**Pull request by drbyte**
I didn't see this addressed elsewhere in the docs.

Is this the best spot for it? I suppose it could go in the Navigation Groups area. 
I'm just not sure where people will "look for" this to be explained, and what words they will use when searching, so that we give them something to find with the search.

### CODE REVIEW

**Comment by drbyte** [comment]
Example from the Demo site:
Screen Shot 2024-01-02 at 10 43 37 PM

**Comment by zepfietje** [comment]
It's not just for group items, also for nested items of a parent which has an icon and children without.

**Comment by danharrin** [comment]
But that one isn't a choice, so I don't think there is much to explain in the docs.

**Comment by zepfietje** [comment]
> But that one isn't a choice, so I don't think there is much to explain in the docs.

Fair.

File: packages/panels/docs/06-navigation.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ To customize a navigation item's [icon](https://blade-ui-kit.com/blade-icons?set
 protected static ?string $navigationIcon = 'heroicon-o-document-text';
 ```
 
+If you set `$navigationIcon = null` on multiple resources within the same navigation Group, those items will be joined with a vertical bar below the Group name.
```
Comment (danharrin) [inline packages/panels/docs/06-navigation.md]:
```suggestion
If you set `$navigationIcon = null` on all items within the same navigation group, those items will be joined with a vertical bar below the Group name.
```

**Pull request merged**